clusters/publick8s.yaml

```
@@ -234,3 +234,13 @@ releases:
       - public-nginx-ingress/public-nginx-ingress
     values:
       - "../config/ipv6-lb-service.yaml"
+  - name: uc-r2
+    namespace: uc-r2
+    chart: jenkins-infra/mirrorbits
+    version: 0.52.0
+    timeout: 600
+    atomic: false
+    values:
+      - "../config/uc-r2.yaml"
```

```suggestion
      - "../config/updates.jenkins.io.yaml"
```

Nitpicking but:
- `updates.jenkins.io` is easier to understand than `uc` as the "Update Center" could mean different things
- R2 is not related to this installation: as we discussed together, R2 is for the controller mirrors